x/payment/types/events.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	EventTypePayForMessage = "payformessage"
+
+	AttributeKeySpender = "spender"
+	AttributeKeySize    = "size"
+)
+
+//NewPayForMessageEvent construt a new payformessge sdk.Event
+func NewPayforMessageEvent(spender sdk.AccAddress, size uint64) sdk.Event {
```

Comment:
@evan-forbes here in `types/events.go`, I created a new event that contains the spender and the size

Reply:
see above comment, we should just a take a string for `spender`. 

as a slight nit pick, I think using a different name than spender would be more consistent with what is used in MsgPayForMessage and MsgWirePayForMessage, which is `signer`. 

another nit pick, also to stay consistent, let's capitalize the f in `NewPayForMessageEvent` :slightly_smiling_face:

Reply:
yeah my bad, it is a typo in the function name :) Thanks @evan-forbes